src/main/resources/META-INF/hudson.remoting.ClassFilter

```diff
@@ -7,3 +7,4 @@ org.bouncycastle.asn1.DERNull
 org.bouncycastle.asn1.DERBitString
 org.bouncycastle.asn1.pkcs.PrivateKeyInfo
 org.bouncycastle.asn1.DEROctetString
+org.bouncycastle.crypto.CipherParameters
```

Comment:
Hmm, interesting. Does this fix the issue with bouncycastle then? (i.e,. the rest reduces to this)

Reply:
This is my attempt to fix it at the time I raised #17, which AFAICS did not do so (builds fail with my version of this plugin for the reason linked in #17).

Reply:
I was able to get it working without the line, I generated an in-toto occurrence and pushed to our test grafeas server. Removing and updating the PR...